types/font-carrier/class/font.d.ts

```diff
@@ -52,5 +54,8 @@ export class Font extends Base<FontOptions> {
      * 导出字体
      * @param options
      */
-    output(options?: FileOptions): Buffer[];
+    output(options?: Font.FileOptions): Buffer[];
 }
+
+export = Font;
+export as namespace Font;
```

Comment:
```suggestion
```
ditto about global script

Reply:
hello,  i have delete it, and commit yesterday